browsers/nodejs.json

```diff
@@ -12,7 +12,7 @@
         },
         "4": {
           "release_date": "2015-09-08",
-          "status": "current"
+          "status": "retired"
```

Comment:
Actual retirement date is April 30th - probably close enough. We may want retirement dates in the JSON in the future.